### PR TITLE
feat: Add translation ability to context methods

### LINKED ISSUE #2083: new feature with translate
in https://github.com/Codeception/CodeceptJS/tree/master/translations is possible add translate.

In translate file is something as:
module.exports = {
  I: 'Ich',
  actions: {
    waitForText: 'warte_auf_text',
    amOnPage: 'bin_auf_seite',
    click: 'clicke',

Please enable translate too:
Scenario, Features.

Maybe will good enable translate commands and config.
For example
, commands: {
    run: 'translateForRun',
    steps: 'translateForSteps',
   generate: 'translateForGenerate'
},
config: {
   helpers: 'translateForHelpers',
   include: 'translateForInclude',
   teardown: 'translateForTeardown',

},
warningsFromAssets: {
}

etc.

### PR DESCRIPTION
**Pull request by koushikmohan1996**
### Motivation/Description of the PR
Added support to translate Feature, Scenario, Before, After, BerforeSuite and AfterSuite

Fixes #2083 

## Type of change

- [ ] Breaking changes
- [X] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [X] Tests have been added
- [X] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [X] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/data/sandbox/configs/translation/translation_test.js  1
- test/runner/translation_test.js  1
         

Clones added
============
- test/runner/translation_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4838770&bid=16068291)

File: translations/it-IT.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 module.exports = {
   I: 'io',
+  contexts: {
+    Feature: 'Caratteristica',
+    Scenario: 'lo_scenario',
+    Before: 'Prima',
+    After: 'Dopo',
+    BeforeSuite: 'Prima_della_suite',
+    AfterSuite: 'Dopo_la_suite',
```
Comment (PeterNgTr) [inline translations/it-IT.js]: may we follow `CamelCase` of js 😄

File: translations/it-IT.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+  },
   actions: {
     amOutsideAngularApp: 'sono_fuori_dalla_applicazione_angular',
     amInsideAngularApp: 'sono_dentro_la_applicazione_angular',
```
Comment (PeterNgTr) [inline translations/it-IT.js]: would be nice if you consider changing them elsewhere 👍

File: translations/it-IT.js (hunk 1 above)
Comment (koushikmohan1996) [reply translations/it-IT.js]: Other translation followed like this in these files, So I thought we are following this pattern :)

File: translations/it-IT.js (hunk 2 above)
Comment (koushikmohan1996) [reply translations/it-IT.js]: Sorry, Did not understand that properly. Can you please elaborate? :)

File: lib/ui.js [hunk 3]
```diff
@@ -6,6 +6,20 @@ const scenario = require('./scenario');
 const ScenarioConfig = require('./interfaces/scenarioConfig');
 const FeatureConfig = require('./interfaces/featureConfig');
 const addDataContext = require('./data/context');
+const container = require('./container');
+
+
+const setContextTranslation = (context) => {
```
Comment (PeterNgTr) [inline lib/ui.js]: what do think if we rename it to `passedContext`? It's kinda confusing when you see context and contexts. That's easy to mess them up :)

Comment (koushikmohan1996) [reply lib/ui.js]: Valid point! will do that :)

File: translations/it-IT.js (hunk 1 above)
Comment (PeterNgTr) [reply translations/it-IT.js]: ah okay then, cause `CamelCase` would make us look like js ^^

File: translations/it-IT.js (hunk 2 above)
Comment (PeterNgTr) [reply translations/it-IT.js]: I meant the same `CamelCase` changing. But yeah, depends on how it benefits.

Comment (koushikmohan1996) [reply translations/it-IT.js]: Changing this now will break the backward compatibility : (

Comment (PeterNgTr) [reply translations/it-IT.js]: that would be `3.0` then, breaking things are tough! ^^

**Pull request merged**